addons/mail/static/src/components/discuss/discuss_tests.js

```diff
@@ -5519,6 +5519,288 @@ QUnit.test('moderation: sent message rejected in moderated channel', async funct
     );
 });
 
+QUnit.test('moderation: as moderator, pending moderation message accessibility', async function (assert) {
```

Comment:
This file is too big, we should split it. We could start now by creating one for the moderation features.

Reply:
yeah but how to name the file and where to place it .. ?

Reply:
> yeah but how to name the file and where to place it .. ?

place the file in same directory as `discuss_tests.js` and name it `discuss_moderation_tests.js`.

Reply:
Don't we want the tests subfolder if we have many tests?